repos/fdbt-site/src/data/auroradb.ts

```diff
@@ -155,6 +156,34 @@ export const getServicesByNocCodeAndDataSource = async (nocCode: string, source:
     }
 };
 
+export const checkBodshasServicesByNoc = async (nationalOperatorCode: string): Promise<boolean> => {
+    const nocCodeParameter = replaceInternalNocCode(nationalOperatorCode);
+
+    logger.info('', {
+        context: 'data.auroradb',
+        message: 'retrieving number of services for given national operator code',
+        noc: nationalOperatorCode,
+    });
+
+    try {
+        const queryInput = `
+        SELECT count(id) as serviceCount
+        FROM txcOperatorLine
+        WHERE nocCode = ?  AND dataSource = 'bods'
+        ORDER BY CAST(lineName AS UNSIGNED), lineName;
+        `;
+
+        const queryResults = await executeQuery<ServiceCount[]>(queryInput, [nocCodeParameter]);
+        const hasService = queryResults[0].serviceCount ? queryResults[0].serviceCount : 0;
```

Comment:
I think we can create a boolean here. `const hasService = queryResults[0].serviceCount !== 0;`

Reply:
there is one already